roles/keycloak_quarkus/tasks/restart.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: "Restart and enable {{ keycloak.service_name }} service"
+  throttle: 1
```

Comment:
Should we consider adding this to other use of systemd (in other collections)?

Reply:
yep